src/gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -18240,19 +18287,33 @@ properties:
         - Leptomeninges
         - Liver
         - Lower lobe, lung
+        - Lower Third Esophagus
```

Comment:
This seems to not meet the current convention (see Lymph Node). Recommend changing to `Esophagus, Lower Third` so that they group together alphabetically.

Reply:
sgtm 👍